packages/node_modules/@node-red/editor-client/locales/en-US/editor.json

```diff
@@ -887,5 +887,10 @@
             "unexpected": "An unexpected error occurred",
             "code": "code"
         }
+    },
+    "editor-tab": {
+        "properties": "@Properties",
```

Comment:
Extraneous `@` symbols in these messages

Reply:
Sorry.  Removed `@`.
Added Japanese translation of new messages.